src/test/java/pages/HomePage.java

```diff
@@ -12,15 +11,15 @@
 import java.time.Duration;
 import java.util.List;
 
-public class HomePage extends BasePage{
+public class HomePage extends BasePage {
 
-    @FindBy(css = "[data-testid='sidebar-create-playlist-btn']")
+    @FindBy(css = "[data-testid=sidebar-create-playlist-btn]")
     WebElement createPlaylistButton;
 
-    @FindBy(css = "[data-testid='playlist-context-menu-create-simple']")
+    @FindBy(css = "[data-testid=playlist-context-menu-create-simple]")
```

Comment:
**I recommend:  make all the locators and WebElements private**, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

_For example_

```
   @FindBy(css = "[data-testid=sidebar-create-playlist-btn]")
    private WebElement createPlaylistButton;

    @FindBy(css = "[data-testid=playlist-context-menu-create-simple]")
    private WebElement newPlaylistOption;

    @FindBy (css = "[name=name]")
    private WebElement playlistNameField;

    @FindBys(   @FindBy(css = "#playlists a[href*='playlist']")  )
    
    private List<WebElement> playlists;
```